bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/AngularVelocity.scala

```diff
@@ -0,0 +1,82 @@
+package edu.gemini.spModel.core
+
+import scalaz._, Scalaz._
+
+/** An angular velocity in [mas/y] in the range [-1296000000, 1296000000] equivalent to [-360, 360] degrees. */
+sealed trait AngularVelocity extends java.io.Serializable {
```

Comment:
The reason maybe not be very good. Without the limit it won't be fully compatible with the old world angle, I hit that on the tests but it could be simpler to limit the range on the tests